capture.go

```diff
@@ -0,0 +1,84 @@
+package main
+
+import (
+	"time"
+
+	log "github.com/Sirupsen/logrus"
+	"github.com/google/gopacket"
+	"github.com/google/gopacket/layers"
+	"github.com/google/gopacket/pcap"
+)
+
+type tcpCapture struct {
```

Comment:
might be better to have an interface here, e.g. we might want to capture udp packets to